### ISSUE TITLE
Videos not getting trained

### ISSUE THREAD

**Issue by D3voz** [kohya-ss/musubi-tuner#73]
Hi, i am not getting any good results from video training. 

I used 15 videos all at 3sec, 24fps. 

This is the dataset.toml- 

[general]
resolution = [854, 480]
caption_extension = ".txt"
batch_size = 4
enable_bucket = true
bucket_no_upscale = false

[[datasets]]
video_directory = "E:\\musubi-tuner\\indata"
cache_directory = "E:\\musubi-tuner\\indata\\cache"
target_frames = [73]
frame_extraction = "head"

And -   
accelerate launch --num_cpu_threads_per_process 1 --mixed_precision bf16 hv_train_network.py --dit E:\\ComfyUI\\models\\diffusion_models\\mp_rank_00_model_states.pt --dataset_config E:\\musubi-tuner\\dataset.toml --sdpa --mixed_precision bf16 --optimizer_type adamw8bit --learning_rate 5e-5 --gradient_checkpointing --max_data_loader_n_workers 2 --persistent_data_loader_workers --network_module networks.lora --network_dim 32 --timestep_sampling shift --discrete_flow_shift 7.0 --max_train_epochs 300 --save_every_n_epochs 50 --seed 42 --output_dir E:\\musubi-tuner\\output --output_name SSJ1

The results without the trained loras are better than the results with the lora.

Link for this lora if anyone wants to check - https://huggingface.co/mdevoz/ssj1_test

https://github.com/user-attachments/assets/b407d64e-6c75-4647-9801-1faf8be3c1d2

**Comment by Sarania** [comment]
I'm guessing you're just way under trained. You are using an LR of 5e-5 and a batch size of 4 with 15 inputs across 300 epochs. 4 batches per epoch would equate to 1200 total steps which is just not enough at that LR. I usually target around 2000 total steps when running LR 2e-4 and a batch size of 1. Larger batch size will need higher LR too. My suggestion would be try a run with batch_size=1 and LR = 2e-4, and --max_train_steps 2000 instead of --max_train_epochs 300. Hope this helps!

**Comment by D3voz** [comment]
Hi, thanks.  Will try it .
